routers/admin/users.go

```diff
@@ -205,6 +205,14 @@ func EditUserPost(ctx *context.Context, form auth.AdminEditUserForm) {
 		u.EncodePasswd()
 	}
 
+	if u.IsLocal() && u.Name != form.UserName {
```

Comment:
`u.IsLocal() && len(form.UserName) > 0 && u.Name != form.UserName` just to be safe. That way you don't need the `binding:"required"` above :)